lefthook.yaml

```diff
@@ -16,4 +16,4 @@ commit-msg:
       run: bun commitlint --edit {1}
 
     spell-check:
-      run: bun cspell --no-summary {1}
+      run: bun cspell -c ./cspell.config.mts --no-summary {1}
```

Comment:
Probably `bun spell-check --no-summary {1}` also works to re-use the config from the package.json scripts.